build.scollector.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
```

Comment:
Yes, but this is triggered by our (StackExchange's) build scripts, which require it to be at the project root.